### PR TITLE
Merge updates

### PR DESCRIPTION
**Pull request by James-Tobin**
Fixes issues such as #61 (Graduate Student not being a real option) and issue where deny_team_in_top_ten didn't actually stop anything.

### CODE REVIEW

**Comment by Bialogs** [comment]
Can we get a description for this pr?

**Comment by Bialogs** [comment]
Can you add a test to check that teams NOT in the top 10 can update their team? I know we have [this](https://github.com/James-Tobin/registration-app/blob/25779827c92319d55f668387a4ff633f877dc68c/test/controllers/teams_controller_test.rb#L160-L166) but I want to be explicit and have a test 
1) Where game is closed and updates are allowed (which I think is what I linked above) 
2) Game is open and top 10 is locked and 
3) Game is open and not in top 10 is not locked.

**Comment by James-Tobin** [comment]
@Bialogs If I understand how rails testing works (which there is a good chance I don't), your link is actually for 3 since in fixtures the game is set to be currently happening.  I should be able to make a test for 1 though.

File: test/controllers/teams_controller_test.rb [hunk 1]
```diff
@@ -181,5 +181,14 @@ class TeamsControllerTest < ActionController::TestCase
     assert_equal I18n.t('teams.in_top_ten'), flash[:alert]
   end
 
+  test 'can not update team when in top ten' do
```
Comment (rbclark) [inline test/controllers/teams_controller_test.rb]: Could you change this to say `cannot update team when game is open and team in top ten`

**Pull request merged**